From 95d9867ce45f4e964026812ba420afe6e492cb13 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 26 Jun 2026 23:14:03 +0800
Subject: [PATCH] AOSCOS: qml: drop KNewStuff functions/buttons

The "Get new ..." button may sound like a good idea, but in reality may
cause issue as plugins may be shipped as binaries, and in worse cases,
contain malware.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 src/ui/main.qml | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/ui/main.qml b/src/ui/main.qml
index e70d8f5..903efd5 100644
--- a/src/ui/main.qml
+++ b/src/ui/main.qml
@@ -11,18 +11,8 @@ import QtQuick.Window 2.2
 import QtQuick.Controls 2.3 as QtControls
 import org.kde.kirigami 2.4 as Kirigami
 import org.kde.kcmutils as KCM
-import org.kde.newstuff 1.91 as NewStuff
 
 KCM.GridViewKCM {
-    actions: NewStuff.Action {
-        id: newStuffButton
-        text: i18nc("@action:button as in, get new Plymouth boot splash screens", "Get New…")
-        enabled: !kcm.busy
-        icon.name: "get-hot-new-stuff"
-        configFile: "plymouth.knsrc"
-        onEntryEvent: (entry, event) =>  kcm.onEntryEvent(entry);
-    }
-
     headerPaddingEnabled: false // Let the InlineMessage touch the edges
     header: Kirigami.InlineMessage {
         id: infoLabel
-- 
2.52.0

